wf/__init__.py

```diff
@@ -0,0 +1,415 @@
+from typing import Annotated, List, Optional
+
+from flytekit.core.annotation import FlyteAnnotation
+from latch.resources.launch_plan import LaunchPlan
+from latch.resources.workflow import workflow
+from latch.types import metadata
+from latch.types.directory import LatchDir, LatchOutputDir
+from latch.types.file import LatchFile
+
+from wf.entrypoint import (
+    DADATaxonomy,
+    Kraken2RefTaxonomy,
+    QiimeRefTaxonomy,
+    SampleSheet,
+    SidleRefTaxonomy,
+    SintaxRefTaxonomy,
+    initialize,
+    nextflow_runtime,
+)
+
+
+@workflow(metadata._nextflow_metadata)
+def nf_nf_core_ampliseq(
+    input_source: str,
+    run_name: Annotated[
+        str,
+        FlyteAnnotation(
+            {
+                "rules": [
+                    {
+                        "regex": r"^[a-zA-Z0-9_-]+$",
+                        "message": "ID name must contain only letters, digits, underscores, and dashes. No spaces are allowed.",
+                    }
+                ],
+            }
+        ),
```

Comment:
this can be added to the NextflowMetadata like this https://wiki.latch.bio/workflows/sdk/nextflow/interface/param-validation

prefer this because it easier to understand compared to adding flyte annotations